mailipy/gen.py

```diff
@@ -79,6 +75,13 @@ def main():
     if not os.path.exists(args.outbox):
         os.mkdir(args.outbox)
 
+    # jinja2 with builtin support (e.g. zip, len, max, ...)
+    env = jinja2.Environment(loader=jinja2.BaseLoader)
+    env.globals.update(__builtins__)
+
+    def render_template(template, data):
```

Comment:
Why move this function inside the main? 🤔

Reply:
To avoid constructing the `Environment` for each mail. The function is inside for capturing `env`.